demo/admin/codegen.ts

```diff
@@ -1,4 +1,10 @@
 import { CodegenConfig } from "@graphql-codegen/cli";
+import { readFileSync } from "fs";
+import { buildSchema } from "graphql";
+
+const schema = buildSchema(readFileSync("./schema.gql").toString());
+
+const rootBlocks = Object.keys(schema.getTypeMap()).filter((type) => type.endsWith("BlockData") || type.endsWith("BlockInput"));
```

Comment:
This code needs to be added to all our applications (both admin and site). Should we move it into a package? Could be either `@comet/cli`,  or `@comet/cms-admin` and `@comet/cms-site`.

Reply:
Reading the schema, extracting the root blocks and generating the scalars object and import statement

Reply:
hm... I'm not sure it's worth it

`@comet/cli` seems wrong (it's not a cli) - I would have chosen a different name for the package anyway :)

Reply:
> hm... I'm not sure it's worth it

Okay, we'll add it to the apps for now.

> I would have chosen a different name for the package anyway :)

You could have said something back then 😉 How'd you name it? `@comet/utils`?